### DISCUSSION TITLE
关于文档中「targetAccount 与 defaultCashAccount 的增减账户关系」的表格字段调整的建议

### DISCUSSION THREAD

**Discussion by ramsayleung** [deb-sig/double-entry-generator#103] [General]
README 中有关于 支付宝, 微信及中国工商银行的 `targetAccount` 与 `defaultCashAccount` 的增减账户的说明:
> `targetAccount` 与 `defaultCashAccount` 的增减账户关系如下表：

|收/支 |defaultCashAccount |targetAccount |
|:-- |:-- |:-- |
| 收入| plusAccount|minusAccount |
| 支出| minusAccount|plusAccount |

但是从逻辑上讲, `defaultCashAccount` 和 `targetAccount` 是由用户配置的值，是属于变量; 而 `plusAccount`和 `minusAccount` 属于每笔交易的固有属性,  因此将固有属性`plusAccount` 和 `minusAccount` 作为表头, 变量作为随「收/支」而变化的值会让逻辑更清晰?

|收/支 |plusAccount |minusAccount |
|:-- |:-- |:-- |
| 收入| defaultCashAccount|targetAccount |
| 支出| targetAccount|defaultCashAccount |

**Comment by Triple-Z** [comment]
LGTM
